backend/src/main/java/com/htecgroup/skynest/controller/BucketController.java

```diff
@@ -164,9 +165,9 @@ public ResponseEntity<BucketResponse> createBucket(
                   examples = {@ExampleObject(value = "Internal Server Error")})
             })
       })
-  @GetMapping("/{uuid}")
+  @GetMapping("/{uuid}/info")
   public ResponseEntity<BucketResponse> getBucket(@PathVariable UUID uuid) {
-    BucketResponse bucketResponse = bucketService.getBucket(uuid);
+    BucketResponse bucketResponse = bucketService.getBucketDetails(uuid);
```

Comment:
You can change it to bucketId, since we are changing every endpoint.

Reply:
Of course.